<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <!-- Load our pregrasp motion -->
  <rosparam command="load" file="$(find manipulation)/config/motions/tiago_motions.yaml"/>
  <!-- Load definition of pregrasp motion into play_motion namespace -->
  <rosparam command="load" file="$(find tiago_pick_demo)/config/pick_motions.yaml" /> 

  <!-- Pick & place server cheezitz-->  
  <node name="pick_and_place_server" pkg="tiago_pick_demo" type="pick_and_place_server.py" output="screen">
      <rosparam command="load" file="$(find tiago_pick_demo)/config/pick_and_place_params.yaml" />
      <param name="object_width"  value="0.158" />
      <param name="object_height" value="0.210" />
      <param name="object_depth"  value="0.06" />
  </node>

  <!-- Pick & place server coke can-->
  <!-- <node name="pick_and_place_server" pkg="tiago_pick_demo" type="pick_and_place_server.py" output="screen">
      <rosparam command="load" file="$(find tiago_pick_demo)/config/pick_and_place_params.yaml" />
      <param name="object_width"  value="0.07" />
      <param name="object_height" value="0.12" />
      <param name="object_depth"  value="0.07" />
  </node> -->

  <!-- Node exposing service to start looking for the object and trigger the picking -->

  <node name="grasp_service" pkg="manipulation" type="manipulate.py" output="screen"/>
  <node name="octomap_rethrottle" pkg="topic_tools" type="throttle" args="messages /xtion/depth_registered/points 2  /throttle_filtering_points/filtered_points" />
  <!-- <node name="look_around" pkg="play_motion" type="run_motion" args="head_tour" launch-prefix="bash -c 'sleep 15.0; $0 $@' "/> -->

</launch>
